web/styles/search.css

```diff
@@ -244,16 +248,16 @@
         .search_list_item .pill-container {
             margin-left: 5px;
         }
-    }
 
-    .pill {
-        align-items: baseline;
-    }
+        .pill {
+            align-items: baseline;
+        }
 
-    .pill-image {
-        /* Add line-height equal to height to mimic baseline alignment. */
-        line-height: 20px;
-        align-self: center;
+        .pill-image {
+            /* Add line-height equal to height to mimic baseline alignment. */
+            line-height: 20px;
+            align-self: center;
+        }
```

Comment:
Scoped these as styling only for pills within the typeahead -- can make this a separate commit.